packages/sdk-js/src/types/growthbook.ts

```diff
@@ -114,7 +114,9 @@ export type Experiment<T> = {
   groups?: string[];
 };
 
-export type AutoExperiment = Experiment<AutoExperimentVariation> & {
+export type AutoExperiment<T = AutoExperimentVariation> = Experiment<T> & {
+  uid?: string;
```

Comment:
What if we call this changeId or changesetId or something like that. Experiment id already means something (the exp_ id used in Mongo and the rest API).

Reply:
sure, called it `changeId`